cmd/openshift-install/create.go

```diff
@@ -410,6 +410,12 @@ func waitForBootstrapComplete(ctx context.Context, config *rest.Config) *cluster
 		timeout = 60 * time.Minute
 	}
 
+	if platformName == aws.Name && installConfig.Config.Platform.AWS != nil && installConfig.Config.Platform.AWS.UserProvisionedDNS == dns.UserProvisionedDNSEnabled {
+		timeout = 60 * time.Minute
+		logrus.Infof("With AWS userProvisionedDNS enabled, waiting up to %v (until %v %s) for bootstrapping to complete...",
```

Comment:
nit: We also have a similar log on line 421 [here](https://github.com/openshift/installer/pull/9953/files#diff-56276d5381d618d46ec8d35d93210662c8fdd4c9bcd90fd36afbc6a59227eb0bR421), so with the changes, we have duplicate info log:

```
INFO With AWS userProvisionedDNS enabled, waiting up to %v (until %v %s) for bootstrapping to complete...
INFO Waiting up to %v (until %v %s) for bootstrapping to complete...
```

How about making it a debug log?

```suggestion
		logrus.Debugf("With AWS userProvisionedDNS enabled, setting bootstrap timeout to %v", timeout)
```